docs/app/Components/ComponentDoc/ComponentExample/ComponentExample.js

```diff
@@ -109,9 +109,8 @@ class ComponentExample extends Component {
   }
 
   handleDirectLinkClick = () => {
-    const { location } = this.props
     this.setHashAndScroll()
-    copyToClipboard(location.href)
+    copyToClipboard(window.location.href)
```

Comment:
We made push to history in `setHashAndScroll`, so we can simply grab `href` from `window.location`